.flowconfig

```diff
@@ -83,7 +83,6 @@ untyped-type-import=warn
 nonstrict-import=warn
 deprecated-type=warn
 unsafe-getters-setters=warn
-inexact-spread=warn
```

Comment:
Here's the explanation in the linked RN commit https://github.com/facebook/react-native/commit/87370c4c0 :
> After changes to the semantics of spread, this lint is no longer relevant and will be removed in an upcoming version of Flow.

Based on that, I think ideally we'd remove this after the upgrade -- that way we have the lint as long as we're still on versions of Flow where it might be relevant.

Is it causing an error or warning at the upgrade? If so it'd be fine to remove it before, but probably I'd still want it as part of the same PR/branch as the upgrade.

... OK, I went and did a quick history search (`git log --stat -p -S inexact-spread --pickaxe-all`) in the Flow repo, and it looks like
* the lint was outright removed, so it probably becomes an error to have it in .flowconfig, in v0.117 at https://github.com/facebook/flow/commit/759970c1b6dc0a25171d9969e34eaf5dae70f130;
* the lint had already stopped doing anything in v0.111 at https://github.com/facebook/flow/commit/0410e050246f43d280a793c8a153930390c3b6a7 -- or at least, it had stopped ever firing in any situation exercised by Flow's tests. (The later commit that removed it didn't have to touch the tests, except to remove the lint from a flowconfig.)

We're on v0.113.0, in between those -- so in fact now is a perfect time to remove this lint from our flowconfig.

(Seems like an oversight that this isn't mentioned in the Flow changelog, at https://github.com/facebook/flow/blob/master/Changelog.md#01170 .)